qtpy/Qt3DAnimation.py

```diff
@@ -4,18 +4,34 @@
 # Licensed under the terms of the MIT License
 # (see LICENSE.txt for details)
 # -----------------------------------------------------------------------------
-"""Provides Qt3DAnimation classes and functions."""
 
-# Local imports
-from . import PYQT5, PYSIDE2, PythonQtError, PYSIDE_VERSION
+"""
+Provides Qt3DAnimation classes and functions.
+"""
+
+from . import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PythonQtError, QtModuleNotInstalledError, API_NAME
 
 if PYQT5:
-    from PyQt5.Qt3DAnimation import *
+    try:
+        from PyQt5.Qt3DAnimation import *
+    except ModuleNotFoundError as error:
+        raise QtModuleNotInstalledError(name='Qt3DAnimation', binding=API_NAME, missing_package='PyQt3D')
```

Comment:
```suggestion
        raise QtModuleNotInstalledError(
            name='Qt3DAnimation', binding=API_NAME, missing_package='PyQt3D'
        ) from error
```

Example of `raise from`

Reply:
Thank you, I was focusing on getting the new args right and forgot that this was RE-raising an exception. I'll fix this everywhere else, too.

Reply:
Sorry, because of stupid effin' GitHub this got randomly submitted silently separate from, and well before, my actual in progress review, but fortunately it seems you're well aware of it and it was simply a minor oversight.